src/ESPiLight.cpp

```diff
@@ -429,3 +449,67 @@ int ESPiLight::stringToPulseTrain(const String &data, uint16_t *codes,
   }
   return -1;
 }
+
+static protocols_t *find_proto(const char *name) {
+  struct protocols_t *pnode = protocols;
+  while (pnode != nullptr) {
+    if (strcmp(name, pnode->listener->id) == 0) {
```

Comment:
`strcmp()` should commonly avoided. I would prefer some more C++ style or Arduino `String`. Otherwise use `strncmp()`?

Reply:
As we don't know how long both of the strings are in advance and rely on them being trminated, it doesn't matter here. 

The Arduino `String` would just add more heap overhead and does a `strcmp()` in the end anyway: https://github.com/arduino/Arduino/blob/master/hardware/arduino/avr/cores/arduino/WString.cpp#L441